task: Task set line length linting for ruff to 170

### Component

API Server / GraphQL, Git Integration, Python SDK, infrahubctl CLI

### Task Description

The target line-length for our Python code is currently 120 but a number of lines are longer than that. In order to pass current checks the line-length in pyproject.toml is set to 215. As an initial step to shorten these long lines we can set it to 170.

Current violations:
```
❯ ruff check .
backend/infrahub/api/diff.py:16:38: TCH001 Move application import `infrahub.message_bus.rpc.InfrahubRpcClient` into a type-checking block
backend/infrahub/api/schema.py:11:34: TCH001 Move application import `infrahub.core.branch.Branch` into a type-checking block
backend/infrahub/api/transformation.py:18:38: TCH001 Move application import `infrahub.message_bus.rpc.InfrahubRpcClient` into a type-checking block
backend/infrahub/auth.py:7:32: TCH002 Move third-party import `starlette.requests.HTTPConnection` into a type-checking block
backend/infrahub/cli/git_agent.py:8:22: TCH002 Move third-party import `aio_pika.IncomingMessage` into a type-checking block
backend/infrahub/core/attribute.py:5:18: TCH003 Move standard library import `uuid.UUID` into a type-checking block
backend/infrahub/core/branch.py:41:38: TCH001 Move application import `infrahub.message_bus.rpc.InfrahubRpcClient` into a type-checking block
backend/infrahub/core/manager.py:6:18: TCH003 Move standard library import `uuid.UUID` into a type-checking block
backend/infrahub/core/manager.py:8:21: TCH002 Move third-party import `graphql.GraphQLSchema` into a type-checking block
backend/infrahub/core/node/__init__.py:4:18: TCH003 Move standard library import `uuid.UUID` into a type-checking block
backend/infrahub/core/query/relationship.py:9:25: TCH002 Move third-party import `neo4j.graph.Relationship` into a type-checking block
backend/infrahub/core/registry.py:8:39: TCH001 Move application import `infrahub.core.definitions.Brancher` into a type-checking block
backend/infrahub/core/relationship.py:4:18: TCH003 Move standard library import `uuid.UUID` into a type-checking block
backend/infrahub/core/schema.py:8:31: TCH002 Move third-party import `typing_extensions.Self` into a type-checking block
backend/infrahub/core/timestamp.py:5:31: TCH002 Move third-party import `pendulum.datetime.DateTime` into a type-checking block
backend/infrahub/git/handlers.py:26:29: TCH002 Move third-party import `infrahub_client.InfrahubClient` into a type-checking block
backend/infrahub/git/repository.py:11:18: TCH003 Move standard library import `uuid.UUID` into a type-checking block
backend/infrahub/graphql/__init__.py:6:21: TCH002 Move third-party import `graphql.GraphQLSchema` into a type-checking block
backend/infrahub/graphql/app.py:25:8: TCH002 Move third-party import `graphene` into a type-checking block
backend/infrahub/graphql/app.py:39:5: TCH002 Move third-party import `graphql.language.ast.DocumentNode` into a type-checking block
backend/infrahub/graphql/app.py:40:5: TCH002 Move third-party import `graphql.language.ast.OperationDefinitionNode` into a type-checking block
backend/infrahub/graphql/app.py:45:19: TCH002 Move third-party import `neo4j.AsyncSession` into a type-checking block
backend/infrahub/graphql/app.py:50:29: TCH002 Move third-party import `starlette.types.Receive` into a type-checking block
backend/infrahub/graphql/app.py:50:38: TCH002 Move third-party import `starlette.types.Scope` into a type-checking block
backend/infrahub/graphql/app.py:50:45: TCH002 Move third-party import `starlette.types.Send` into a type-checking block
backend/infrahub/graphql/app.py:68:34: TCH001 Move application import `infrahub.core.branch.Branch` into a type-checking block
backend/infrahub/graphql/mutations.py:7:19: TCH002 Move third-party import `neo4j.AsyncSession` into a type-checking block
backend/infrahub/graphql/mutations.py:26:38: TCH001 Move application import `infrahub.message_bus.rpc.InfrahubRpcClient` into a type-checking block
backend/infrahub/graphql/schema.py:6:21: TCH002 Move third-party import `graphql.GraphQLResolveInfo` into a type-checking block
backend/infrahub/graphql/utils.py:3:8: TCH003 Move standard library import `abc` into a type-checking block
backend/infrahub/message_bus/rpc.py:9:5: TCH002 Move third-party import `aio_pika.abc.AbstractChannel` into a type-checking block
backend/infrahub/message_bus/rpc.py:10:5: TCH002 Move third-party import `aio_pika.abc.AbstractIncomingMessage` into a type-checking block
backend/infrahub/message_bus/rpc.py:11:5: TCH002 Move third-party import `aio_pika.abc.AbstractQueue` into a type-checking block
backend/infrahub/message_bus/rpc.py:12:5: TCH002 Move third-party import `aio_pika.abc.AbstractRobustConnection` into a type-checking block
python_sdk/infrahub_client/node.py:9:36: TCH001 Move application import `infrahub_client.schema.AttributeSchema` into a type-checking block
python_sdk/infrahub_client/node.py:9:53: TCH001 Move application import `infrahub_client.schema.NodeSchema` into a type-checking block
python_sdk/infrahub_client/node.py:9:65: TCH001 Move application import `infrahub_client.schema.RelationshipSchema` into a type-checking block
Found 37 errors.
```

As a reference these are the lines that currently are above 120 characters:

```
❯ ruff check .
backend/infrahub/api/background.py:47:121: E501 Line too long (150 > 120 characters)
backend/infrahub/api/diff.py:180:121: E501 Line too long (134 > 120 characters)
backend/infrahub/api/diff.py:216:121: E501 Line too long (151 > 120 characters)
backend/infrahub/api/diff.py:298:121: E501 Line too long (134 > 120 characters)
backend/infrahub/checks.py:44:121: E501 Line too long (126 > 120 characters)
backend/infrahub/cli/git_agent.py:107:121: E501 Line too long (130 > 120 characters)
backend/infrahub/core/account.py:36:121: E501 Line too long (139 > 120 characters)
backend/infrahub/core/account.py:39:121: E501 Line too long (141 > 120 characters)
backend/infrahub/core/attribute.py:357:121: E501 Line too long (123 > 120 characters)
backend/infrahub/core/branch.py:272:121: E501 Line too long (151 > 120 characters)
backend/infrahub/core/branch.py:305:121: E501 Line too long (131 > 120 characters)
backend/infrahub/core/branch.py:361:121: E501 Line too long (132 > 120 characters)
backend/infrahub/core/branch.py:401:121: E501 Line too long (173 > 120 characters)
backend/infrahub/core/branch.py:402:121: E501 Line too long (125 > 120 characters)
backend/infrahub/core/branch.py:430:121: E501 Line too long (147 > 120 characters)
backend/infrahub/core/branch.py:431:121: E501 Line too long (191 > 120 characters)
backend/infrahub/core/branch.py:836:121: E501 Line too long (140 > 120 characters)
backend/infrahub/core/branch.py:933:121: E501 Line too long (122 > 120 characters)
backend/infrahub/core/manager.py:342:121: E501 Line too long (137 > 120 characters)
backend/infrahub/core/manager.py:434:121: E501 Line too long (122 > 120 characters)
backend/infrahub/core/manager.py:690:121: E501 Line too long (122 > 120 characters)
backend/infrahub/core/manager.py:846:121: E501 Line too long (143 > 120 characters)
backend/infrahub/core/query/diff.py:111:121: E501 Line too long (141 > 120 characters)
backend/infrahub/core/query/diff.py:112:121: E501 Line too long (187 > 120 characters)
backend/infrahub/core/query/diff.py:168:121: E501 Line too long (159 > 120 characters)
backend/infrahub/core/query/diff.py:171:121: E501 Line too long (184 > 120 characters)
backend/infrahub/core/query/node.py:291:121: E501 Line too long (146 > 120 characters)
backend/infrahub/core/relationship.py:564:121: E501 Line too long (136 > 120 characters)
backend/infrahub/core/relationship.py:684:121: E501 Line too long (126 > 120 characters)
backend/infrahub/core/schema.py:66:121: E501 Line too long (126 > 120 characters)
backend/infrahub/core/schema.py:175:121: E501 Line too long (124 > 120 characters)
backend/infrahub/core/schema.py:535:121: E501 Line too long (122 > 120 characters)
backend/infrahub/git/handlers.py:172:121: E501 Line too long (122 > 120 characters)
backend/infrahub/git/handlers.py:185:121: E501 Line too long (135 > 120 characters)
backend/infrahub/git/repository.py:77:121: E501 Line too long (133 > 120 characters)
backend/infrahub/git/repository.py:83:121: E501 Line too long (126 > 120 characters)
backend/infrahub/git/repository.py:86:121: E501 Line too long (123 > 120 characters)
backend/infrahub/git/repository.py:101:121: E501 Line too long (125 > 120 characters)
backend/infrahub/git/repository.py:103:121: E501 Line too long (169 > 120 characters)
backend/infrahub/git/repository.py:256:121: E501 Line too long (121 > 120 characters)
backend/infrahub/git/repository.py:377:121: E501 Line too long (141 > 120 characters)
backend/infrahub/git/repository.py:697:121: E501 Line too long (125 > 120 characters)
backend/infrahub/git/repository.py:741:121: E501 Line too long (141 > 120 characters)
backend/infrahub/git/repository.py:755:121: E501 Line too long (123 > 120 characters)
backend/infrahub/git/repository.py:807:121: E501 Line too long (142 > 120 characters)
backend/infrahub/git/repository.py:961:121: E501 Line too long (124 > 120 characters)
backend/infrahub/git/repository.py:1074:121: E501 Line too long (130 > 120 characters)
backend/infrahub/graphql/app.py:2:121: E501 Line too long (124 > 120 characters)
backend/infrahub/graphql/app.py:643:121: E501 Line too long (123 > 120 characters)
backend/infrahub/graphql/generator.py:385:121: E501 Line too long (124 > 120 characters)
backend/infrahub/graphql/utils.py:34:121: E501 Line too long (124 > 120 characters)
backend/infrahub/graphql/utils.py:226:121: E501 Line too long (136 > 120 characters)
backend/infrahub/transforms.py:43:121: E501 Line too long (126 > 120 characters)
backend/tests/integration/user_workflows/test_user_worflow.py:100:121: E501 Line too long (124 > 120 characters)
backend/tests/integration/user_workflows/test_user_worflow.py:265:121: E501 Line too long (121 > 120 characters)
backend/tests/integration/user_workflows/test_user_worflow.py:665:121: E501 Line too long (121 > 120 characters)
backend/tests/unit/api/conftest.py:12:121: E501 Line too long (135 > 120 characters)
backend/tests/unit/core/test_attribute.py:151:121: E501 Line too long (141 > 120 characters)
backend/tests/unit/core/test_attribute.py:159:121: E501 Line too long (141 > 120 characters)
backend/tests/unit/core/test_attribute.py:169:121: E501 Line too long (132 > 120 characters)
backend/tests/unit/core/test_branch.py:86:121: E501 Line too long (143 > 120 characters)
backend/tests/unit/core/test_branch.py:87:121: E501 Line too long (145 > 120 characters)
backend/tests/unit/core/test_branch.py:88:121: E501 Line too long (143 > 120 characters)
backend/tests/unit/core/test_branch.py:89:121: E501 Line too long (145 > 120 characters)
backend/tests/unit/core/test_node_query.py:114:121: E501 Line too long (129 > 120 characters)
backend/tests/unit/core/test_schema.py:285:121: E501 Line too long (215 > 120 characters)
backend/tests/unit/core/test_schema.py:294:121: E501 Line too long (132 > 120 characters)
backend/tests/unit/git/conftest.py:211:121: E501 Line too long (123 > 120 characters)
python_sdk/infrahub_client/client.py:225:121: E501 Line too long (122 > 120 characters)
python_sdk/infrahub_client/client.py:227:121: E501 Line too long (148 > 120 characters)
python_sdk/infrahub_client/client.py:504:121: E501 Line too long (122 > 120 characters)
python_sdk/infrahub_client/client.py:506:121: E501 Line too long (148 > 120 characters)
python_sdk/infrahub_client/node.py:489:121: E501 Line too long (127 > 120 characters)
python_sdk/infrahub_client/node.py:579:121: E501 Line too long (131 > 120 characters)
python_sdk/infrahub_client/schema.py:130:121: E501 Line too long (122 > 120 characters)
python_sdk/infrahub_client/schema.py:175:121: E501 Line too long (129 > 120 characters)
python_sdk/infrahub_client/store.py:29:121: E501 Line too long (122 > 120 characters)
python_sdk/tests/unit/test_node.py:533:121: E501 Line too long (126 > 120 characters)
sync/diffsync/diffsync/__init__.py:148:121: E501 Line too long (125 > 120 characters)
sync/diffsync/diffsync/__init__.py:176:121: E501 Line too long (121 > 120 characters)
sync/diffsync/diffsync/__init__.py:202:121: E501 Line too long (134 > 120 characters)
sync/diffsync/diffsync/__init__.py:240:121: E501 Line too long (134 > 120 characters)
sync/diffsync/diffsync/__init__.py:269:121: E501 Line too long (134 > 120 characters)
sync/diffsync/diffsync/__init__.py:554:121: E501 Line too long (130 > 120 characters)
sync/diffsync/diffsync/__init__.py:678:121: E501 Line too long (122 > 120 characters)
sync/diffsync/diffsync/__init__.py:693:121: E501 Line too long (122 > 120 characters)
sync/diffsync/diffsync/__init__.py:815:121: E501 Line too long (126 > 120 characters)
sync/diffsync/diffsync/__init__.py:820:121: E501 Line too long (132 > 120 characters)
sync/diffsync/diffsync/helpers.py:399:121: E501 Line too long (122 > 120 characters)
sync/diffsync/diffsync/store/__init__.py:45:121: E501 Line too long (122 > 120 characters)
sync/diffsync/diffsync/store/__init__.py:54:121: E501 Line too long (121 > 120 characters)
sync/diffsync/diffsync/store/__init__.py:68:121: E501 Line too long (121 > 120 characters)
sync/diffsync/diffsync/store/__init__.py:104:121: E501 Line too long (121 > 120 characters)
sync/diffsync/diffsync/store/__init__.py:183:121: E501 Line too long (126 > 120 characters)
sync/diffsync/diffsync/store/__init__.py:188:121: E501 Line too long (132 > 120 characters)
sync/diffsync/diffsync/store/local.py:41:121: E501 Line too long (122 > 120 characters)
sync/diffsync/diffsync/store/local.py:56:121: E501 Line too long (121 > 120 characters)
sync/diffsync/diffsync/store/local.py:75:121: E501 Line too long (121 > 120 characters)
sync/diffsync/diffsync/store/redis.py:90:121: E501 Line too long (122 > 120 characters)
sync/diffsync/diffsync/store/redis.py:103:121: E501 Line too long (121 > 120 characters)
sync/diffsync/diffsync/store/redis.py:126:121: E501 Line too long (121 > 120 characters)
sync/infrahub-sync/infrahub_sync/netbox/client.py:70:121: E501 Line too long (148 > 120 characters)
tasks/performance.py:23:121: E501 Line too long (124 > 120 characters)
Found 103 errors.
```

--- a/backend/infrahub/core/branch.py
+++ b/backend/infrahub/core/branch.py
@@ -371,8 +371,12 @@ class Branch(StandardNode):
             for idx, branch_name in enumerate(start_times.keys()):
                 filters_per_rel.extend(
                     [
-                        f"({rel}.branch = $branch{idx} AND {rel}.from >= $start_time{idx} AND {rel}.from <= $end_time{idx} AND ( r2.to is NULL or r2.to >= $end_time{idx}))",
-                        f"({rel}.branch = $branch{idx} AND {rel}.from >= $start_time{idx} AND {rel}.to <= $start_time{idx})",
+                        f"""({rel}.branch = $branch{idx}
+                             AND {rel}.from >= $start_time{idx}
+                             AND {rel}.from <= $end_time{idx}
+                             AND ( r2.to is NULL or r2.to >= $end_time{idx}))""",
+                        f"""({rel}.branch = $branch{idx} AND {rel}.from >= $start_time{idx}
+                            AND {rel}.to <= $start_time{idx})""",
                     ]
                 )
 
@@ -399,8 +403,11 @@ class Branch(StandardNode):
         params["end_time"] = end_time.to_string()
 
         filters_per_rel = [
-            f"({rel_label}.branch in $branches AND {rel_label}.from >= $start_time AND {rel_label}.from <= $end_time AND {rel_label}.to IS NULL)",
-            f"({rel_label}.branch in $branches AND (({rel_label}.from >= $start_time AND {rel_label}.from <= $end_time) OR ({rel_label}.to >= $start_time AND {rel_label}.to <= $end_time)))",
+            f"""({rel_label}.branch in $branches AND {rel_label}.from >= $start_time
+                 AND {rel_label}.from <= $end_time AND {rel_label}.to IS NULL)""",
+            f"""({rel_label}.branch in $branches AND (({rel_label}.from >= $start_time
+                 AND {rel_label}.from <= $end_time) OR ({rel_label}.to >= $start_time
+                 AND {rel_label}.to <= $end_time)))""",
         ]
 
         filters.append("(" + "\n OR ".join(filters_per_rel) + ")")

--- a/backend/infrahub/core/query/diff.py
+++ b/backend/infrahub/core/query/diff.py
@@ -107,8 +107,15 @@ class DiffRelationshipQuery(DiffQuery):
     async def query_init(self, session: AsyncSession, *args, **kwargs):
         query = """
         MATCH p = ((sn)-[r1]->(rel:Relationship)<-[r2]-(dn))
-        WHERE (r1.branch = r2.branch AND (r1.to = r2.to OR (r1.to is NULL AND r2.to is NULL)) AND r1.from = r2.from AND r1.status = r2.status
-          AND all(r IN relationships(p) WHERE (r.branch IN $branch_names AND r.from >= $diff_from AND r.from <= $diff_to AND ((r.to >= $diff_from AND r.to <= $diff_to) OR r.to is NULL))))
+        WHERE (r1.branch = r2.branch AND
+            (r1.to = r2.to OR
+                (r1.to is NULL AND r2.to is NULL)
+            ) AND r1.from = r2.from AND r1.status = r2.status
+        AND all(r IN relationships(p) WHERE
+            (r.branch IN $branch_names AND r.from >= $diff_from AND r.from <= $diff_to
+                AND ((r.to >= $diff_from AND r.to <= $diff_to) OR r.to is NULL))
+            )
+        )
         """
 
         self.add_to_query(query)
@@ -164,10 +171,12 @@ class DiffRelationshipPropertyQuery(DiffQuery):
         query = (
             """
         MATCH (rel:Relationship)-[r3:IS_VISIBLE|IS_PROTECTED|HAS_SOURCE|HAS_OWNER]-(rp)
-        WHERE (r3.branch IN $branch_names AND r3.from >= $diff_from AND r3.from <= $diff_to AND ((r3.to >= $diff_from AND r3.to <= $diff_to) OR r3.to is NULL))
+        WHERE (r3.branch IN $branch_names AND r3.from >= $diff_from AND r3.from <= $diff_to
+            AND ((r3.to >= $diff_from AND r3.to <= $diff_to) OR r3.to is NULL))
         WITH *
         MATCH p = ((sn)-[r1]->(rel)<-[r2]-(dn))
-        WHERE r1.branch = r2.branch AND (r1.to = r2.to OR (r1.to is NULL AND r2.to is NULL)) AND r1.from = r2.from AND r1.status = r2.status AND all(r IN relationships(p) WHERE ( %s ))
+        WHERE r1.branch = r2.branch AND (r1.to = r2.to OR (r1.to is NULL AND r2.to is NULL))
+            AND r1.from = r2.from AND r1.status = r2.status AND all(r IN relationships(p) WHERE ( %s ))
         """
             % rels_filter
         )

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -190,7 +190,7 @@ select = [
     "W",
 ]
 
-line-length = 215
+line-length = 170
 
 [tool.ruff.mccabe]
 # Target max-complexity=10
@@ -199,7 +199,7 @@ max-complexity = 33
 [tool.ruff.per-file-ignores]
 
 "backend/infrahub/git/repository.py" = [
-    "TCH003", # Pydantic needs UUID import to not only be available under YPE_CHECKING clause
+    "TCH003", # Pydantic needs UUID import to not only be available under TYPE_CHECKING clause
 ]
 
 [build-system]

